tests/installation/test_executor.py

```diff
@@ -153,26 +153,23 @@ def callback(
 
 
 @pytest.fixture()
-def copy_wheel(tmp_dir: Path) -> Callable[[], Path]:
+def copy_wheel(tmp_path: Path) -> Callable[[], Path]:
     def _copy_wheel() -> Path:
         tmp_name = tempfile.mktemp()
-        Path(tmp_dir).joinpath(tmp_name).mkdir()
+        tmp_path.joinpath(tmp_name).mkdir()
 
         shutil.copyfile(
             Path(__file__)
             .parent.parent.joinpath(
                 "fixtures/distributions/demo-0.1.2-py2.py3-none-any.whl"
             )
             .as_posix(),
-            Path(tmp_dir)
-            .joinpath(tmp_name)
+            tmp_path.joinpath(tmp_name)
             .joinpath("demo-0.1.2-py2.py3-none-any.whl")
             .as_posix(),
         )
 
-        return (
-            Path(tmp_dir).joinpath(tmp_name).joinpath("demo-0.1.2-py2.py3-none-any.whl")
-        )
+        return tmp_path.joinpath(tmp_name).joinpath("demo-0.1.2-py2.py3-none-any.whl")
```

Comment:
I think is more readable write:

`tmp_path / tmp_name / "demo-0.1.2-py2.py3-none-any.whl"`